discord/types/subscription.py

```diff
@@ -40,3 +40,4 @@ class Subscription(TypedDict):
     current_period_end: str
     status: SubscriptionStatus
     canceled_at: Optional[str]
+    renewal_sku_ids: List[Snowflake]
```

Comment:
```suggestion
    renewal_sku_ids: Optional[List[Snowflake]]
```

Kind of weird this is considered a required key on their end but this is more accurate I guess.